### ISSUE TITLE
LDAP "ca" directive not set in master-config.yml when openshift_master_ldap_ca_file is configured

### ISSUE THREAD

**Issue by davidaah** [openshift/openshift-ansible#6191]
#### Description

When setting `openshift_master_ldap_ca_file` I found that /etc/origin/master/master-config.yml does not contain the ldap_ca.crt directive.

https://docs.openshift.com/container-platform/3.6/install_config/configuring_authentication.html#LDAPPasswordIdentityProvider

The issue appears to be that openshift_master_ldap_ca_file is mapped to `ldap_ca` variable which is never passed to the [ldap provider generator](https://github.com/openshift/openshift-ansible/blob/release-3.6/roles/openshift_master_facts/filter_plugins/openshift_master.py#L158)

##### Version

```
openshift-ansible-3.6.173.0.31-1
```

##### Steps To Reproduce
1. Configure provider and certificate

openshift_master_identity_providers=[{'name': 'active_directory', 'challenge': 'true', 'login': 'true', 'kind': 'LDAPPasswordIdentityProvider', 'attributes': {'id': ['...'], 'email': ['...'], 'name': ['...'], 'preferredUsername': ['...']}, 'bindDN': '...', 'bindPassword': '...', 'insecure': 'false', 'url': '...'}]

openshift_master_ldap_ca_file=/etc/ansible/ldap-ca-certificates/tls.crt

2. Run installation

##### Expected Results
Expected block in /etc/origin/master/master-config.yml

```
oauthConfig:
   ...
  grantConfig:
    method: auto
  identityProviders:
  - challenge: true
    login: true
    mappingMethod: claim
    name: active_directory
    provider:
      apiVersion: v1
      attributes:
        email:
        - ...
        id:
        - ...
        name:
        - ...
        preferredUsername:
        - ...
      bindDN: ...
      bindPassword: ...
      insecure: false
      ca: ldap_ca.crt
      kind: LDAPPasswordIdentityProvider
      url: ...
```

##### Observed Results

```
oauthConfig:
   ...
  grantConfig:
    method: auto
  identityProviders:
  - challenge: true
    login: true
    mappingMethod: claim
    name: active_directory
    provider:
      apiVersion: v1
      attributes:
        email:
        - ...
        id:
        - ...
        name:
        - ...
        preferredUsername:
        - ...
      bindDN: ...
      bindPassword: ...
      insecure: false
      kind: LDAPPasswordIdentityProvider
      url: ...
```

```
`ca` directive was absent causing validation errors when talking to LDAP server
```

**Comment by davidaah** [comment]
This is an issue with 3.7 as well. a quick and dirty fix was to add `ca: ldap_ca.crt` to the identity provider config which is the default value for LDAPPasswordIdentityProvider in the advanced installer

```
 identityProviders:
  - challenge: true
    login: true
    mappingMethod: claim
    name: active_directory
    ca: ldap_ca.crt
    provider:
      apiVersion: v1
      attributes:
        email:
        - ...
        id:
        - ...
        name:
        - ...
        preferredUsername:
        - ...
      bindDN: ...
      bindPassword: ...
      insecure: false
      kind: LDAPPasswordIdentityProvider
      url: ...
```

**Comment by vrutkovs** [comment]
This is fixed in  #9731 - `ca:` is ignored now, instead in 3.10+ the installer would copy `_ldap_ca_file` to `/etc/origin/master/<name>_ldap_ca.crt`
